### DISCUSSION TITLE
3,5" LCD Touch ILI9481 + ESP32 Wroom

### DISCUSSION THREAD

**Discussion by DSRocker** [HASwitchPlate/openHASP#269] [Hardware]
Hi,
I would like to use my 3,5" Touch Display with openHASP. Is it possible right now or in future?
Thanks a lot for this good joob! 👍

**Comment by fvanroie** [comment]
ILI9481 displays are supported via the TFT_eSPI or LovyanGFX libraries.
Can you provide a link to the product specifications of the screen and ESP32 you are using?

**Comment by DSRocker** [comment]
I have this one: https://electropeak.com/learn/interfacing-3-5-inch-full-color-touch-tft-display-module-with-arduino/

**Reply by DSRocker** [reply]
ESP32 Dev Kit C
I 2 versions:
https://www.az-delivery.de/en/products/esp32-dev-kit-c-v4-unverlotet
https://www.az-delivery.de/en/products/esp32-developmentboard

**Reply by fvanroie** [reply]
> I have this one: https://electropeak.com/learn/interfacing-3-5-inch-full-color-touch-tft-display-module-with-arduino/

That display should work using this [contributed configuration file](https://github.com/HASwitchPlate/openHASP/blob/master/user_setups/esp32/d1-r32-unoshield_ili9486_adc.ini). It does not have a touch controller, so the touches are sampled using ADC instead. Also this screen does not support backlight control. I would advise using the [Waveshare Touch Shield](https://haswitchplate.github.io/openHASP-docs/0.6.2/displays/waveshare-tft-touch-shield/) instead.

> ESP32 Dev Kit C

These dev boards are typically used in the [AZ-Touch enclosure](https://www.az-delivery.de/en/products/az-touch-wandgehauseset-mit-2-8-zoll-touchscreen-fur-esp8266-und-esp32) or [FreeTouchDeck](https://haswitchplate.github.io/openHASP-docs/latest/devices/freetouchdeck/#hasp-build_flags) using the combiner PCB provided.
Of course, you can also connect the display using jumper cables according to the configuration file.
